code/style.css

```diff
@@ -1,5 +1,47 @@
-/* Here in the css file you'll write your styling. make sure to link the css file to the html file. If it's linked correctly you should see a light blue background. */
+body     { font-family: "Gill Sans Extrabold", Helvetica, sans-serif }
 
-body {
-  background-color: lightblue;
+.receipt { font-family: Courier, "Lucida Console", monospace }
```

Comment:
Your code will look neater if you write it like this:

.receipt {
  font-family: Courier, "Lucida Console", monospace;
}

And don't forget the semicolons at the end of css statements. ✔️